ui/src/components/CreateOrEditTask.vue

```diff
@@ -241,6 +241,17 @@ export default {
       loading: {
         createTask: false,
       },
+      localuser: "",
```

Comment:
JS variables should use camelCase, `localuser`  and the following lines use no case or snake_case

Reply:
should be good @andre8244